store-implementation/parquet-store/src/test/java/uk/gov/gchq/gaffer/parquetstore/operation/handler/spark/AbstractSparkOperationsTest.java

```diff
@@ -137,14 +139,21 @@ public void getElementsWithSeedsRelatedAfterImportElementsFromRDDTestWhenMoreFil
         ElementUtil.assertElementEquals(getResultsForGetElementsWithSeedsRelatedTest(), results);
     }
 
-//    @Test
-//    public void getDataFrameOfElementsTest() throws IOException, OperationException, StoreException {
-//        final Graph graph = genData(false);
-//        final Dataset<Row> data = graph.execute(new GetDataFrameOfElements.Builder()
-//                .build(), user);
-//        checkGetDataFrameOfElements(data, false);
-//    }
-//
+    @Test
+    public void getDataFrameOfElementsTest() throws OperationException {
+        // Given
+        final Graph graph = createGraph();
+        final List<Element> elements = getInputDataForGetAllElementsTestAsList();
+        graph.execute(new AddElements.Builder().input(elements).build(), user);
+
+        // When
+        final Dataset<Row> results = graph.execute(new GetDataFrameOfElements.Builder().build(), user);
+
+        // Then
+        final List<Element> elementsFromRows = convertRowsToElements(results.collectAsList());
+        ElementUtil.assertElementEquals(getResultsForGetAllElementsTest(), elementsFromRows);
+    }
+
 //    @Test
```

Comment:
Can you remove these commented out tests

Reply:
Done, although they weren't added in this PR.

Reply:
These tests are still there in the final version of the PR.